get_involved.md

```diff
@@ -25,7 +25,7 @@ Announcements of Beta and RCs happen through
 
 ## Contribute as a Developer
 
-See the [developers guide](http://docs.pulpproject.org/dev-guide/contributing/index.html) for how
+See the [developers guide](http://docs.pulpproject.org/en/2.21/dev-guide/contributing/index.html) for how
```

Comment:
Again, if 3.0 is preferred here, I can update to https://docs.pulpproject.org/contributing/index.html

Reply:
+1